dll/win32/shell32/shelldesktop/CDesktopBrowser.cpp

```diff
@@ -349,9 +359,15 @@ HRESULT STDMETHODCALLTYPE CDesktopBrowser::SetToolbarItems(LPTBBUTTON lpButtons,
     return E_NOTIMPL;
 }
 
+HRESULT STDMETHODCALLTYPE CDesktopBrowser::GetPropertyBag(long flags, REFIID riid, void **ppv)
+{
+    DWORD deskpidl = 0;
+    return SHGetViewStatePropertyBag((LPITEMIDLIST)&deskpidl, L"Desktop", flags | SHGVSPB_ROAM, riid, ppv);
```

Comment:
```suggestion
    ITEMIDLIST deskpidl = {0};
    return SHGetViewStatePropertyBag(&deskpidl, L"Desktop", flags | SHGVSPB_ROAM, riid, ppv);
```
to avoid the ugly cast.